webapp/src/ts/components/duplicate-contacts/duplicate-contacts.component.html

```diff
@@ -1,20 +1,20 @@
 <mat-expansion-panel
-  (opened)="_loadContactSummary()"
-  [hideToggle]="isLoading">
+  (opened)="loadContactSummary()"
+  [hideToggle]="isLoading"
+  *ngIf="duplicate">
   <mat-expansion-panel-header>
-    <mat-panel-title> <strong>{{ 'duplicate_check.contact.name'  | translate }} </strong> &nbsp;{{ duplicate?.name }} </mat-panel-title>
-    <mat-panel-description> <strong>{{ 'duplicate_check.contact.created_on' | translate }} </strong> &nbsp;{{ duplicate?.reported_date | simpleDate }} </mat-panel-description>
+    <mat-panel-title> <strong>{{ 'duplicate_check.contact.name'  | translate }} </strong> &nbsp;{{ duplicate.name }} </mat-panel-title>
+    <mat-panel-description> <strong>{{ 'duplicate_check.contact.created_on' | translate }} </strong> &nbsp;{{ duplicate.reported_date | simpleDate }} </mat-panel-description>
   </mat-expansion-panel-header>
   <div class="nested-section">
     <div class="loader inline small" *ngIf="isLoading"></div>
-    <p *ngIf="error" class="error_msg"><strong>{{ error!.toString() }}</strong></p>
+    <p *ngIf="error" class="loading-status"><strong>{{ error }}</strong></p>
     <mm-contact-summary-content
-      [fields]="_summary?.fields"
+      [fields]="contactSummary?.fields"
       [contactsLoadingSummary]="isLoading">
     </mm-contact-summary-content>
   </div>
   <mat-action-row>
-    <button class="btn submit btn-primary" *ngIf="error" (click)="_loadContactSummary()">{{ 'duplicate_check.contact.retry' | translate }}</button>
```

Comment:
Happy to discuss this further, but after thinking about this more, I did not really see the point of having the "Retry" button.  The contact-summary calculation is not really the kind of thing that we expect to fail sometimes, but may work if you try it again. If it fails once, it will likely continue to fail.

So, instead of a giving a Retry button, I just made the "Open" button always visible. If the contact-summary fails, the user will see a message indicating it failed to load, but they can still use the "Open" button to navigate to the contact if they want.